pkg/trace/api/endpoints.go

```diff
@@ -88,6 +89,13 @@ var endpoints = []endpoint{
 		Pattern: "/profiling/v1/input",
 		Handler: func(r *HTTPReceiver) http.Handler { return r.profileProxyHandler() },
 	},
+	{
+		Pattern: "/telemetry/proxy/",
+		Handler: func(r *HTTPReceiver) http.Handler {
```

Comment:
Can you please make this function all on one line? All of the others are too.

Reply:
Its automatically expanded for me to 3 lines because of its length. Are there any line limits ?

Reply:
It's fine then. Leave it as it is.